src/views/Package/PackageHeader/Heading.tsx

```diff
@@ -14,6 +14,10 @@ import { CDKTypeIcon, CDKTypeText } from "../../../components/CDKType";
 import { PackageTag } from "../../../components/PackageTag";
 import { KEYWORD_IGNORE_LIST } from "../../../constants/keywords";
 
+interface TagObject extends PackageTagConfig {
+  isKeyword?: boolean;
+}
```

Comment:
I wonder if we can avoid redefining this interface in multiple places (it's also used in Tags.tsx). The rest of this file also has a lot of similar code that looks as though it can be reused / extracted

Reply:
Yeah the whole flow for rendering tags / keywords is an opportunity for code-reuse that I can add in a follow-up!